apps/src/applab/DesignToolboxElement.jsx

```diff
@@ -45,37 +45,37 @@ export default React.createClass({
         </div>
       </div>
     );
-  },
+  }
 
-  componentDidMount: function () {
+  componentDidMount() {
     this.makeDraggable();
-  },
+  }
 
-  componentDidUpdate: function () {
+  componentDidUpdate() {
     this.makeDraggable();
-  },
+  }
```

Comment:
seems like these need to be bound since they access `this`, is that correct?

Reply:
Lifecycle methods don't need to be bound - React calls them in the component's context already.